recipes/ci-sdr/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ci-sdr" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ci_sdr-{{ version }}.tar.gz
+  sha256: a1387f39ccd55cce034e2c01000a0a337b3729d8a5010b42c5381d8c820fa4bb
+
+build:
+  skip: true  # [win]
```

Comment:
The `pytorch` runtime dependency does not have Windows builds. There are compatible Windows builds in the `pytorch` channel, but not in the `conda-forge` channel.

Reply:
Our policy for noarch is that if all platforms have the same dependencies, but dependencies are missing because they are not available for some platforms, this package should still be built as noarch. In this way, if pytorch is built for Windows in the future, this package will become available for Windows too without you needing to do anything.